src/Components/Nav/Nav.js

```diff
@@ -0,0 +1,117 @@
+import { Link, useLocation } from 'react-router-dom';
+import { withRouter } from 'react-router-dom/cjs/react-router-dom.min';
+import styled from 'styled-components';
+
+function Nav() {
+  let locatioin = useLocation();
+
+  return (
+    <NavWrap>
+      <Logo location={locatioin} to="/">
```

Comment:
```suggestion
  const isMainPage = location.pathname === '/'

  return (
    <NavWrap>
      <Logo isActivate={isMainPage} to="/">
```
이렇게 구현하셔도 기능은 동작하겠지만 제 생각에는 상태 로직을 컴포넌트 안에서 작성해주고 그 상태에 따라 스타일드 컴포넌트가 변하도록 해보면 어떨까 싶어요! 위와 같은 느낌만 전달드리니 세부 구현은 상황에 맞게 바꿔주세요!

Reply:
수정했는데 바뀐 코드를 말씀하신게 맞나요?👀